admin/worker/deployments_health_check.go

```diff
@@ -179,15 +179,32 @@ func (w *Worker) deploymentHealthCheck(ctx context.Context, d *database.Deployme
 		for k, v := range annotations.ToMap() {
 			f = append(f, zap.String(k, v))
 		}
+
+		// log metrics view errors
+		logger := w.logger.WithLazy(f...)
+		if health.ParseErrorCount > 0 || health.ReconcileErrorCount > 0 {
+			logger.Error("deployment health check: project has parse/reconcile errors", zap.Int32("parse_errors", health.ParseErrorCount), zap.Int32("reconcile_errors", health.ReconcileErrorCount))
+		}
+		for d, err := range health.MetricsViewErrors {
+			logger.Error("deployment health check: dashboard error", zap.String("dashboard", d), zap.String("error", err))
+		}
+
+		onlyUnhealthyDash := true
 		if health.OlapError != "" {
+			onlyUnhealthyDash = false
 			f = append(f, zap.String("olap_error", health.OlapError))
```

Comment:
1. This is just collecting `zap` fields, the actual log is incorporated into `runtime instance is unhealthy` log only. 
2. Done.

Reply:
Okay. Parse and reconcile errors can be clubbed but metrics view errors need to be logged at warn level separately for each metrics view otherwise it losses context that this metrics view has this error. 
I can add a count in the one log line for a quick overview.